e2e/test/scenarios/embedding/sdk-iframe-embedding/custom-elements-api.cy.spec.ts

```diff
@@ -0,0 +1,361 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import {
+  ORDERS_COUNT_QUESTION_ID,
+  ORDERS_DASHBOARD_ID,
+  ORDERS_QUESTION_ID,
+} from "e2e/support/cypress_sample_instance_data";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATABASE;
+
+const { H } = cy;
+
+describe("scenarios > embedding > sdk iframe embedding > custom elements api", () => {
+  beforeEach(() => {
+    cy.signInAsAdmin();
+    H.prepareSdkIframeEmbedTest({
+      withTokenFeatures: true,
+    });
+  });
+
+  describe("<metabase-dashboard>", () => {
+    it("should embed a dashboard with <metabase-dashboard dashboard-id='${number}'>", () => {
+      H.visitCustomHtmlPage(`
+      ${H.getNewEmbedScriptTag()}
+
+      ${H.getNewEmbedConfigurationScript({})}
```

Comment:
tiny nit: we can add a default `{}` to `getNewEmbedConfigurationScript` so we can call `H.getNewEmbedConfigurationScript()` directly